static/app/views/performance/browser/webVitals/utils/useProjectWebVitalsTimeseriesQuery.tsx

```diff
@@ -81,13 +80,20 @@ export const useProjectWebVitalsTimeseriesQuery = ({transaction}: Props) => {
   };
 
   result?.data?.['p75(measurements.lcp)'].data.forEach((interval, index) => {
+    const lcp: number = result?.data?.['p75(measurements.lcp)'].data[index][1][0].count;
+    const fcp: number = result?.data?.['p75(measurements.fcp)'].data[index][1][0].count;
+    const cls: number = result?.data?.['p75(measurements.cls)'].data[index][1][0].count;
+    const ttfb: number = result?.data?.['p75(measurements.ttfb)'].data[index][1][0].count;
+    const fid: number = result?.data?.['p75(measurements.fid)'].data[index][1][0].count;
+    // This is kinda jank, but since events-stats zero fills, we need to assume that 0 values mean no data.
+    // It's unlikely that we'll ever have a 0 value for a web vital, so this should be fine.
```

Comment:
FID can def be 0 - but for now we don't have to worry about that edge case. Maybe useful to try to figure out how big of a problem it could be?

Reply:
yeah this is true. Ive even seen events with 0 cls. This is probably something we'll have to tackle in the backend in the future. Update the comment to make this more clear!